app/packages/core/src/components/Sidebar/ViewSelection/index.tsx

```diff
@@ -134,7 +134,11 @@ export default function ViewSelection(props: Props) {
   }, [searchData, selected]);
 
   const loadedView = useRecoilValue<fos.State.Stage[]>(fos.view);
-  const isEmptyView = !loadedView?.length;
+  const [selectedSavedView] = useRecoilState<DatasetViewOption | null>(
+    selectedSavedViewState
+  );
+  const isEmptyView = false; //selectedSavedView.label !== "Unsaved view"; //
```

Comment:
Same reason as above (to enable saving extended view). However, for the case of saving all samples, I've been debating whether it is worth checking all the additional atom states of the extended view just to prevent people from saving the full view.

On a semi-related note, it feels like there should be a way to get back to a full view from a saved view using the view selector, but right now, the only way to do it is to remove everything from the view bar. This isn't hard or complicated, but doesn't feel as intuitive

Reply:
I think it is ok to initially allow saving all samples and remove that check - we can add a guard later if needed. cc @brimoor to confirm

I agree there should be a better way to get to full view besides clearing the viewbar - we could think through a new option in our saved view selection 🤔

Reply:
remove the unused code